dotCMS/src/integration-test/java/com/dotcms/storage/StoragePersistenceAPITest.java

```diff
@@ -295,7 +338,7 @@ public void Test_Push_Large_File(final TestCase testCase)
         }
 
         return new Object[]{
-                new TestCase(StorageType.FILE_SYSTEM, groupName, path, temp),
+              //  new TestCase(StorageType.FILE_SYSTEM, groupName, path, temp),
```

Comment:
why is the FileSystem storage commented?

Reply:
yeah, I was chasing a bug and I wanted to narrow the scope. restored